src/components/QuizTemplate.tsx

```diff
@@ -132,9 +133,31 @@ const QuizTemplate: React.FC<QuizProps> = QuizProps => {
     return shuffleModalArr[0];
   };
 
-  const checkAnswer = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+  const selectOption = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    // Set the selected option
```

Comment:
comment isn't needed here since the next line of code is explanatory

Reply:
Okay, noted.